pom.xml

```diff
@@ -33,6 +33,11 @@
 			<artifactId>spring-boot-configuration-processor</artifactId>
 			<optional>true</optional>
 		</dependency>
+		<dependency>    
+            <groupId>org.springframework.boot</groupId>    
+            <artifactId>spring-boot-starter-data-jpa</artifactId>    
+            <version>2.2.2.RELEASE</version>    
```

Comment:
Do not mention version for Spring data JPA. Spring boot will automatically take the version. remove this line